doc/api/querystring.md

```diff
@@ -101,6 +101,9 @@ added: v0.1.25
 The `querystring.stringify()` method produces a URL query string from a
 given `obj` by iterating through the object's "own properties".
 
+This method searializes the following types of values passed in `obj`: strings,
```

Comment:
> but from what I understood from the linked source code was that an empty string is returned if the param passed if not of the aforementioned types.

That’s how I read the `stringify` source code, too. How about adding something like `Any other input values will be coerced to empty strings?`